tests/foreman/cli/test_repository.py

```diff
@@ -1073,14 +1074,16 @@ def test_positive_synchronize_auth_puppet_repo(self, repo):
         ),
         indirect=True,
     )
-    def test_positive_synchronize_docker_repo(self, repo):
-        """Check if Docker repository can be created and synced
+    def test_positive_synchronize_docker_repo(self, repo, module_product, module_org):
+        """Check if Docker repository can be created, synced, and deleted
 
         :id: cb9ae788-743c-4785-98b2-6ae0c161bc9a
 
         :parametrized: yes
 
-        :expectedresults: Docker repository is created and synced
+        :expectedresults: Docker repository is created, synced, and deleted
+
+        :BZ: 1810165
```

Comment:
This BZ is a customer scenario, please add `:CustomerScenario: true`

Reply:
OK, done that.